features/ouders.feature

```diff
@@ -110,15 +104,13 @@ Rule: de actuele gegevens van ouders worden geleverd
     | fields              | ouders.ouderAanduiding,ouders.naam.voornamen,ouders.datumIngangFamilierechtelijkeBetrekking |
     Dan heeft de ouder met ouderAanduiding '1' de volgende gegevens
     | naam                                          | waarde     |
-    | type                                          | Ouder      |
     | ouderAanduiding                               | 1          |
     | datumIngangFamilierechtelijkeBetrekking.datum | 2019-03-17 |
```

Comment:
dit is een @gba scenario. de datum wordt niet geformatteerd naar een date, maar blijft "20190317"

Reply:
Aangepast